core/templates/dev/head/pages/exploration-editor-page/statistics-tab/services/learner-action-render.service.spec.ts

```diff
@@ -189,315 +189,21 @@ describe('Learner Action Render Service', function() {
         this.PlaythroughService.recordExplorationQuitAction('stateName2', 120);
 
         var learnerActions = this.PlaythroughService.getPlaythrough().actions;
-        var displayBlocks =
-          this.LearnerActionRenderService.getDisplayBlocks(learnerActions);
-
-        expect(displayBlocks).toEqual([[
-          this.LearnerActionObjectFactory.createNew(
-            'ExplorationStart', {
-              state_name: {
-                value: 'stateName1'
-              }
-            }, 1
-          ),
-          jasmine.any(Object),
-          jasmine.any(Object),
-          this.LearnerActionObjectFactory.createNew(
-            'ExplorationQuit', {
-              state_name: {
-                value: 'stateName2'
-              },
-              time_spent_in_state_in_msecs: {
-                value: 120
-              }
-            }, 1
-          )
-        ]]);
-      });
-
-    it('should split up many learner actions into different display blocks.',
```

Comment:
Please justify explicitly the removal of each removed test. Make sure there is a commensurate replacement somewhere else in this PR.

Reply:
@seanlip Will work on this in the next iteration

Reply:
@seanlip 
I noticed that since we reduced complexity on this service, there isn't much to test in this spec file. There was one missed exposed function, for which I added a test. But there is some complexity within the playthrough modal directive which is untested. But I don't think we normally write unit tests for directives right?  The playthrough modal happy path is anyway covered by protractor tests.

Reply:
Ah ok, that sounds reasonable. Are your services spec files 100% covered by tests? (You can check the karma_coverage_reports directory locally.)

Reply:
Done. Its okay.

![Screenshot 2019-12-19 at 9 42 01 AM](https://user-images.githubusercontent.com/15828797/71144420-f95d0e80-2243-11ea-95d7-3a6cdeaea705.png)